huff_utils/src/sol_interface.rs

```diff
@@ -71,12 +74,18 @@ pub fn gen_sol_interfaces(artifacts: &Vec<Arc<Artifact>>) -> Vec<(PathBuf, &str,
                 ));
             });
 
-            let interface_name =
-                artifact.file.path.split('/').last().unwrap().split('.').next().unwrap();
+            let interface_name = interface.clone().unwrap_or_else(|| {
+                format!(
+                    "I{}",
+                    artifact.file.path.split('/').last().unwrap().split('.').next().unwrap()
+                )
+            });
+            let formatted_str =
+                format!("interface I{} {{\n{}\n}}", interface_name, defs.join("\n"));
```

Comment:
adds an extra `I` to the generated interface that is already added in `huffc.rs` on L220 (assuming no override name was passed)